docs/templates/preprocessing/text.md

```diff
@@ -12,15 +12,15 @@ Split a sentence into a list of words.
 
 - __Arguments__:
     - __text__: str.
-    - __filters__: list (or concatenation) of characters to filter out, such as punctuation. Default: base_filter(), includes basic punctuation, tabs, and newlines.
+    - __filters__: list (or concatenation) of characters to filter out, such as punctuation. Default: '!"#$%&()*+,-./:;<=>?@[\\]^_`{|}~\t\n' , includes basic punctuation, tabs, and newlines.
```

Comment:
https://github.com/iamRusty/keras/blob/master/docs/templates/preprocessing/text.md

It works perfectly